torchci/pages/api/drci/drci.ts

```diff
@@ -194,41 +194,53 @@ export function constructResultsComment(
     const headerPrefix = `## `
     const pendingIcon = `:hourglass_flowing_sand:`
     const successIcon = `:white_check_mark:`
-    const noneFailing = ` No Failures`;
-    const someFailing = `## :x: ${failing} Failures`;
-    const somePending = `, ${pending} Pending`;
-
-    const hasFailing = failing > 0;
-    const hasPending = pending > 0;
-    if (!hasFailing) {
-        output += headerPrefix
-        if (hasPending) {
-            output += pendingIcon
-        } else {
-            output += successIcon
-        }
-
-        output += noneFailing;
-
-        if (hasPending) {
-            output += somePending;
-        }
-
-        output += `\nAs of commit ${sha}:`;
-        output += `\n:green_heart: Looks good so far! There are no failures yet. :green_heart:`;
+    const failuresIcon = `:x:`
+    const noneFailing = `No Failures`
+    const significantFailures = `${failedJobs.length} Significant Failures`
```

Comment:
Nit (`1 Significant Failures` feels weird to me)
```suggestion
    const significantFailures = `${failedJobs.length} Significant Failure{failedJobs.length > 1 ? 's' : ''}`
```